### PR TITLE
Fix CS: Use single quotes where applicable.

### PR DESCRIPTION
**Pull request by dereuromark**
For https://github.com/cakephp/cakephp-codesniffer/pull/307

also fixed some other small issues.

### CODE REVIEW

**Review by ADmad — APPROVED** [review]
Nice

File: templates/element/exception_stack_trace.php [hunk 1]
```diff
@@ -13,6 +13,7 @@
  * @link          https://cakephp.org CakePHP(tm) Project
  * @since         1.3.0
  * @license       https://opensource.org/licenses/mit-license.php MIT License
+ * @var array $trace
```
Comment (othercorey) [inline templates/element/exception_stack_trace.php:16]: This doesn't look right.

File: src/Controller/ControllerFactory.php [hunk 2]
```diff
@@ -108,8 +108,8 @@ public function getControllerClass(ServerRequest $request): ?string
             if ($firstChar !== strtoupper($firstChar)) {
                 deprecationWarning(
                     "The `{$prefix}` prefix did not start with an upper case character. " .
-                    "Routing prefixes should be defined as CamelCase values. " .
-                    "Prefix inflection will be removed in 5.0"
+                    'Routing prefixes should be defined as CamelCase values. ' .
```
Comment (othercorey) [inline src/Controller/ControllerFactory.php:111]: Not sure how I feel about having to perfectly switch when editing/writing multi-line strings.

File: templates/element/exception_stack_trace.php (hunk 1 above)
Comment (dereuromark) [reply templates/element/exception_stack_trace.php:16]: We dont use inline allignment

File: src/Controller/ControllerFactory.php (hunk 2 above)
Comment (ADmad) [reply src/Controller/ControllerFactory.php:111]: For such cases sprintf() could be used to avoid using both type of quotes.

Comment (othercorey) [reply src/Controller/ControllerFactory.php:111]: Sounds reasonable to me.

**Pull request merged**